smarts/core/scenario.py

```diff
@@ -540,8 +528,7 @@ def discover_traffic_histories(scenario_root):
         with open(path, "rb") as f:
             files = pickle.load(f)
             for file_name in files:
-                with open(os.path.join(scenario_root, file_name), "r") as history_file:
-                    traffic_histories.append(json.loads(history_file.read()))
+                traffic_histories.append(os.path.join(scenario_root, file_name))
```

Comment:
Are the iterations of the `for` loop independent, or can be made independent, such that they can be replaced with `map` or other constructs to improve performance and speed?